### PR TITLE
doc: fix kubectl explain for routegroups

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

File: cluster/manifests/skipper/crd.yaml [hunk 1]
```diff
@@ -1,158 +1,209 @@
...
   group: zalando.org
-  version: v1
-  scope: Namespaced
   names:
-    kind: RouteGroup
-    singular: routegroup
-    plural: routegroups
     categories:
     - all
-    shortNames:
```
Comment (mikkeloscar) [inline cluster/manifests/skipper/crd.yaml:15]: @szuecs this unfortunately dropped the `shortNames`. I have made a fix here: https://github.com/szuecs/routegroup-client/pull/8

Comment (szuecs) [reply cluster/manifests/skipper/crd.yaml:15]: Thanks, merged it

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**